app/Http/Requests/LogInFormRequest.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LogInFormRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'email' => 'required|email:rfc,dns',
+            'password' => 'required|string'
+        ];
+    }
+    
+    public function messages()
+    {
+        return [
+            // 'message' => 'Algo ha salido mal',
+            'email.required' => 'Este campo es requerido',
+            'password.required' => 'El campo es requerido',
```

Comment:
Try to be consistent
```suggestion
            'email.required' => 'Este campo es requerido',
            'password.required' => 'Este campo es requerido',
```